packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1346,6 +1347,7 @@
   "Username_wants_to_start_otr_Do_you_want_to_accept": "__username__ wants to start OTR. Do you want to accept?",
   "Users": "Users",
   "Users_in_role": "Users in role",
+  "Users_online": "__total__ users online",
```

Comment:
Another comment on this, can you add the plural and singular versions as well? Examples:
* 1 user online
* 2 users online